### PR TITLE
8.0.0

### PR DESCRIPTION
**Pull request by taylorjcase**
## Description
Updated CirrusMDSDK to 8.0.0

## Motivation and Context
Version 8.0.0 contains multiple improvements to help with ease of integration (see notes Module Stability and CirrusMDSDK code renaming below!) as well as a new patient satisfaction survey (optional), accessibility improvements, and a bug fix.

Module Stability: CirrusMDSDK is now built with [Module Stability](https://swift.org/blog/abi-stability-and-more/)! No more Xcode/Swift version matching. From now on CirrusMDSDK versions will work with all versions of Xcode (12.0 and up) and Swift (5.1 and up). **Important: Please see the notes in README.md on Module Stability as you may need to adjust your project settings or use our workaround if that's not possible.**

Code level renaming of CirrusMDSDK to CirrusMD (at a high level and in Cocoapods still named CirrusMDSDK): For improved code brevity and to match our other SDK offerings (Android, React Native) we have renamed many of the classes, functions, enums, variables, etc. from CirrusMDSDK to CirrusMD. **Important: This will require code changes on the consumer side but they should be very simple (simply removing the SDK part of the name).**

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have added tests to cover my changes.
- [ ] No tests are required for this change.
- [x] My change requires a documentation change.
- [x] My change requires a CHANGELOG update.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,13 +33,16 @@ To run the example project, clone the repo, and run `pod install` from the Examp
 
 ## Requirements
 
-- Required Xcode and Swift versions for each release are documented in the [CHANGELOG](https://github.com/CirrusMD/CirrusMD-iOS-SDK-Example/blob/develop/CHANGELOG.md)
+- Requires Xcode 12.0 or later
+- Requires project Swift version to be 5.1 or later
+- Requires `Build Settings > Build Options > Build Libraries for Distribution` to be set to `Yes`. More information on this requirement and a possible workaround can be found in the [Module Stability Requirement](#module-stability-requirement) documentation.
+- Requires CocoaPods version 1.10.0 or later. This is related to the [Module Stability Requirement](#module-stability-requirement).
```
Comment (taylorjcase) [inline README.md:39]: Updated requirements.

File: README.md [hunk 2]
```diff
@@ -774,21 +779,63 @@ case .valid
...
+    case CirrusMDTokenStateInvalid:
+    case CirrusMDTokenStateUnknown:
+    case CirrusMDTokenStateValid:
 }
 ```
 
-## Troubleshooting
 
-1. If you see the following error in an Objective-C project you likely haven't embedded the Swift Standard Libraries. Ensure `Build Settings > Build Options > Always Embed Swift Standard Libraries` is set to `Yes`.
+## Module Stability Requirement
```
Comment (taylorjcase) [inline README.md:792]: Full details on Module Stability.

File: README.md [hunk 3]
```diff
@@ -227,24 +232,24 @@ If the SDK has been provided with a valid secret and token the SSO user's channe
...
 2. Provide custom views for any or all of the optional delegates.
 3. Set a frame on the view you provide and it will be centered in the yellow area of the screen below.
 
 ##### Swift
 
 ```swift
-// CirrusMDSDKDelegate
+// CirrusMDDelegate
 
 func viewForError(code: CirrusMDSDKResult) -> UIView {
```
Comment (DanielGalarza) [inline README.md]: I forget, did this class get renamed as well? (`CirrusMDSDKResult`)

**Review by DanielGalarza — COMMENTED** [review]
Looks good! Just had one small question

File: README.md (hunk 3 above)
Comment (taylorjcase) [reply README.md]: It did! Good catch.

**Review by DanielGalarza — APPROVED** [review]

**Pull request merged**